examples/plot_compare_reduction.py

```diff
@@ -53,7 +53,7 @@
 digits = load_digits()
 grid.fit(digits.data, digits.target)
 
-mean_scores = np.array(grid.cv_results_['mean_test_score'])
+mean_scores = np.array(grid.cv_results_['mean_test_score'], dtype=np.int)
```

Comment:
Sorry, what? Why do you turn scores into ints?
Also this line is too convoluted anyway, it could be:
```python
mean_scores = grid.cv_results_['mean_test_score'] 
```
which is already an array.